frontend/.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Is this file supposed to be here ?

Reply:
idk, gitignore states that it does but i can remove it

Reply:
but about modular structure i totally agree, but this pr is only for remaking forms as pages and not dialogs. I will move those files into separate module after adding `Restore password` page and setting up auth flow